axelrod/actions.py

```diff
@@ -15,3 +16,18 @@ def flip_action(action: Action) -> Action:
         return Actions.C
     else:
         raise ValueError("Encountered a invalid action.")
+
+
+def str_to_actions(actions: str) -> GeneratorType:
+    """Takes a string like 'cCdD' and returns a generator that yields the appropriate actions."""
+    def make_generator(actions_str):
+        for character in actions_str:
+            as_upper = character.upper()
```

Comment:
`str_to_actions('ccddd')`  and   
`str_to_actions('CCDDD')`   
the lower one looks more natural to all the current axelrod code and evokes the actions clearly.  so it would seem to make sense to enforce that in code.  i like that suggestion.  i'll await a decision.